src/librbd/cache/pwl/LogOperation.cc

```diff
@@ -269,9 +266,9 @@ WriteLogOperationSet::~WriteLogOperationSet() { }
 
 std::ostream &operator<<(std::ostream &os,
                          const WriteLogOperationSet &s) {
-  os << "cell=" << (void*)s.cell << ", "
-     << "extent_ops_appending=[" << s.extent_ops_appending << ", "
-     << "extent_ops_persist=[" << s.extent_ops_persist << "]";
+  os << " cell=" << (void*)s.cell
```

Comment:
Is the space actually needed here?  It looks like it is output as ` " op_set=" << *req.op_set` so I guess not?